src/modules/commentsPopup.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-underscore-dangle */
+const openCommentWindow = (movie) => {
+  const CommentsPopup = document.querySelector('.comments-popup');
+  CommentsPopup.style.display = 'flex';
+
+  CommentsPopup.innerHTML = `<div class="top">
+                // eslint-disable-next-line no-underscore-dangle
+                <img class="movie-image" src="${movie._embedded.show.image.medium}"></img>
```

Comment:
The popup image is not properly styled. could you please fix that? Look at the image below.
![noel image](https://github.com/NoelLincoln/movie-safari/assets/43038485/f2ce68d5-b9bd-4e8c-bba1-d1fb45176b55)